shared/folders/list.desktop.js

```diff
@@ -1,94 +1,92 @@
 // @flow
 import React, {Component} from 'react'
-import type {Props} from './list'
+import ReactDOM from 'react-dom'
+import type {Props, Folder} from './list'
 import type {IconType} from '../common-adapters/icon'
 import {Box, Text, Icon} from '../common-adapters'
 import Row from './row'
 import {globalStyles, globalColors} from '../styles/style-guide'
 
-type State = {
-  showIgnored: boolean
-}
-
 const rowKey = users => users && users.map(u => u.username).join('-')
 
-const Ignored = ({showIgnored, ignored, styles, onToggle, isPublic, onOpen, onClick, onRekey, smallMode}) => {
+const Ignored = ({rows, showIgnored, styles, onToggle, isPublic, onClick}) => {
```

Comment:
I belive flow understands these types as its coming from props it already understands. for example if you do rows.blah() it'll complain that its not a matching type